pkg/model/build.go

```diff
@@ -0,0 +1,46 @@
+package model
+
+type rawBuildParams struct {
+	Image     string   `json:"image,omitempty" yaml:"image,omitempty"`
+	Context   string   `json:"context,omitempty" yaml:"context,omitempty"`
+	File      string   `json:"dockerfile,omitempty" yaml:"dockerfile,omitempty"`
+	Target    string   `json:"target,omitempty" yaml:"target,omitempty"`
+	Args      []string `json:"args,omitempty" yaml:"args,omitempty"`
+	Secrets   []string `json:"secrets,omitempty" yaml:"secrets,omitempty"`
+	CacheFrom []string `json:"cacheFrom,omitempty" yaml:"cacheFrom,omitempty"`
+}
+
+type BuildParams struct {
```

Comment:
is it possible to reuse `BuildInfo`?
https://github.com/okteto/okteto/blob/fd03d2c861691e15320dd99f63c97af18674cda7/pkg/model/dev.go#L116

Reply:
the keys for this struct are different, for example, image is called Name, should i add Image to BuildInfo ?

Reply:
Yeap, let's add `Image`. `Name` and where it's used will be deprecated after the build section is released